Iceberg-UI.package/IceRepositoriesBrowser.class/instance/composePackagesListIn..st

```diff
@@ -14,6 +14,7 @@ composePackagesListIn: composite
 		addSelectionAction: self unloadAndRemovePackageAction;
 		addSelectionAction: self removePackageAction;
 		addSelectionAction: self removePackageFromDiskAction;
+		enableFilter: [ :packageModel :pattern | packageModel package name includesSubstring: pattern caseSensitive: false ];
```

Comment:
using `model package name` doesn't feels right. Can you use `model packageName` ?
(I know, it does not exists... you need to add it)